### PR TITLE
Nuru/beautify start later

### PR DESCRIPTION
**Pull request by DanSallau**
Hi,

Attached contained few changes that should beautify our toggle switch as described in the card here https://trello.com/c/PBLotA4R/55-beautify-start-later-toggle-following-abdo-s-suggestion .  Note: I have commented few css styling instead of deleting . I chose to comment those pending decision and would the needful later.

### CODE REVIEW

**Comment by nuruddeensalihu** [comment]
What happened to travis ? 😉

File: public/styles.css [hunk 1]
```diff
@@ -758,7 +758,10 @@ td.right, th.right {
 td.row-id {
     font-weight: bold;
 }
-
+.row {
```
Comment (borisyankov) [inline public/styles.css]: We already have this style in binary-styles that we use.
Also, almost always use rems for this in this case about 0.5rems

File: public/styles.css [hunk 2]
```diff
@@ -851,14 +854,59 @@ input[type=number]::-webkit-outer-spin-button {
...
 }
 
 .trade-chart {
     height: 250px;
     margin-bottom: 1rem;
 }
-
+.onoffswitch {
+    position: relative; width: 90px;
+    -webkit-user-select:none; -moz-user-select:none; -ms-user-select: none;
```
Comment (borisyankov) [inline public/styles.css]: user-select is already set to none for all elements in binary-styles

File: public/styles.css (hunk 1 above)
Comment (nuruddeensalihu) [inline public/styles.css]: I learnt we already have one. I only extend and modify the top and bottom margins. However, we add a new css class for that if necessary ?

**Pull request merged**